aspnet/signalr/overview/performance/scaleout-with-redis.md

```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
 
 In Hyper-V, add a new VM. In the **Connect Virtual Hard Disk** step, select **Create a virtual hard disk**.
 
-![Window titled New Virtual Machine Wizard. 'Connect Virtual Hard Disk' is highlighted and 'Create a virtual hard disk' is selected.](scaleout-with-redis/_static/image2.png)
+![Screenshot of a window titled New Virtual Machine Wizard and Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
```

Comment:
Let's split this sentence up to.  "Screenshot that shows the New Virtual Machine Wizard. Connect Virtual Hard Disk and Create a virtual hard disk are selected.